src/py/mat3ra/made/tools/build/slab/builders.py

```diff
@@ -34,7 +34,9 @@ def _generate(self, configuration: _ConfigurationType) -> List[_GeneratedItemTyp
             reorient_lattice=True,
             primitive=configuration.make_primitive,
         )
-        raw_slabs = generator.get_slabs()
+        raw_slabs = generator.get_slabs(
```

Comment:
Let's put a comment why this is needed

Reply:
Also: Pymatgen changed signature that fails slab creation on python 3.8

[Fix `lll_reduce` for slab generation (#3927) · materialsproject/pymatgen@6bb2938](https://github.com/materialsproject/pymatgen/commit/6bb293896945106476d6676cb2b65550d791edb5)